serverless-scheduler-proxy/main.go

```diff
@@ -99,6 +100,21 @@ func rewriteBotPubSubURL(c botConfig) func(*http.Request) {
 	}
 }
 
+func rewriteBotContainerURL(c botConfig) func(*http.Request) {
+	return func(req *http.Request) {
+		req.Header.Add("x-github-event", "pubsub.message")
+		parser := func(bodyBytes []byte) (string, string) {
+			var pay PubSubMessage
+			json.Unmarshal(bodyBytes, &pay)
```

Comment:
would change to 

```
if err := json.Unmarshal(bodyBytes, &pay); err != nil {
  log.Printf("error occurred parsing pubsub message: %v\n", err)
}
```

Reply:
Just so that if we do ever get an error unmarshaling the bytes we can diagnose the issue